src/pages/aidataset/[accession_id].astro

```diff
@@ -204,8 +209,55 @@ const images = study.image_uuids.map((uuid) => exports["images"][uuid]);
     <div class="vf-section-header">
       <h3 class="vf-section-header__heading" id="annotations"> Viewable Annotations </h3>
     </div>
-    <div></div>
+    <div>
+      <table id="viewable_annotations" class="display">
+        <thead>
+          <tr>
+            <th >Preview</th>
+            <th>Filename</th>
+            <th>Download Size</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {viewable_annotation_images.map((image) =>
+            <tr>
+              <th>
+                <img class="vf-figure__image" src={image.thumbnail_uri}>
+              </th>
+              <th>{image.name}</th>
+              <th>None</th>
+              <th>
+                <a href={"/image/" + image.uuid}>View</a>
+                Download
+              </th>
+            </tr>
+          )}
+        </tbody>
+      </table>
+      </div>
     <div> </div>
   </section>
+
+
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
```

Comment:
Is jquery required by datatables? The is `document.querySelector()` that does the same thing just dropping a dependency

Reply:
No but I couldn't make the non-jquery version work.